### PR TITLE
Fix test cases

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java [hunk 1]
```diff
@@ -9879,7 +9874,8 @@ public void findExperimentRunsHyperparameterWithSortKeyRepository()
         3,
         response.getExperimentRunsCount());
 
-    hyperparameterFilter = Value.newBuilder().setStringValue("abc").build();
+    // TODO: Currently string value not supported in metrics / hyperparameters
```
Comment (ravishetye) [inline backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java]: the todo is incorrect , we do not support mismatch of datatypes.
We do support if all hyperparameters are string

Comment (ravishetye) [reply backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java]: there should be a test case for filtering on a string hyperpparameter is one does not exist already

**Comment by anandjakhaniya** [comment]
Successfully executed all test cases https://jenkins.dev.verta.ai/job/test/job/integration-mdb/104/

**Comment by ravishetye** [comment]
@ad-47 : can you list out the tests which we have skipped in the PR ddescription

**Review by ravishetye — APPROVED** [review]

**Pull request merged**